c87468732.lua

```diff
@@ -64,8 +70,11 @@ end
 function c87468732.cfilter(c)
 	return c:IsFacedown() or not c:IsType(TYPE_PENDULUM)
 end
-function c87468732.sccon(e)
-	return not Duel.IsExistingMatchingCard(c87468732.cfilter,e:GetHandlerPlayer(),LOCATION_MZONE,0,1,nil)
+function c87468732.sccon1(e)
+	return e:GetHandler():GetSequence()==0 and not Duel.IsExistingMatchingCard(c87468732.cfilter,e:GetHandlerPlayer(),LOCATION_MZONE,0,1,nil)
```

Comment:
`GetSequence` of PZONE will return 6 or 7 in MR3

Reply:
In my opinion,  a card released in 2021 does not have to consider Master Rule 3, which ends in 2017.
We did not consider older rule like priority when Master Rule 3 was still active.

After Master Rule 3 ends, the whole rules have changed multiple times.
Many card descriptions have changed, and even the way to resolve some effects have also changed.
Keeping all scripts in the future compatible with Master Rule 3 is almost impossible, and it could make it difficult to fix the scripts.

If one really wants to play the game under Master Rule 3, one can fork this repo and revert it to 2017 Mar where Master Rule 3 is still active.
I think this repo itself should only take care of the current rules.

Reply:
it can be easily fixed by `c==Duel.GetFieldCard(tp,LOCATION_PZONE,0)`